scm-plugins/scm-hg-plugin/src/main/js/HgConfigurationForm.tsx

```diff
@@ -124,16 +140,19 @@ class HgConfigurationForm extends React.Component<Props, State> {
   };
 
   render() {
+    const { t } = this.props;
     return (
       <div className="columns is-multiline">
         {this.inputField("hgBinary")}
         {this.inputField("encoding")}
-        <div className="column is-half">
-          {this.checkbox("showRevisionInId")}
-        </div>
-        <div className="column is-half">
-          {this.checkbox("enableHttpPostArgs")}
-        </div>
+        <div className="column is-half">{this.checkbox("showRevisionInId")}</div>
+        <div className="column is-half">{this.checkbox("enableHttpPostArgs")}</div>
+        <Button
+          disabled={!this.props.initialConfiguration?._links?.autoConfiguration}
+          action={() => this.triggerAutoConfigure()}
+        >
+          {t("scm-hg-plugin.config.autoConfigure")}
+        </Button>
```

Comment:
```suggestion
        <div className="column is-full">
          <Button
            disabled={!this.props.initialConfiguration?._links?.autoConfiguration}
            action={() => this.triggerAutoConfigure()}
          >
            {t("scm-hg-plugin.config.autoConfigure")}
          </Button>
        </div>
```
The new button is further to the left and falls out of the column layout.